src/main/java/com/blackduck/integration/detect/tool/signaturescanner/operation/CreateScanBatchOperation.java

```diff
@@ -128,6 +131,21 @@ private ScanBatch createScanBatch(
         }
     }
     
+    /**
+     * If we are online and if a user has specified they want csvArchives, warn if the 
+     * BlackDuck server we are connected to can't handle it.
+     */
+    private void attemptToSetCsvArchive(ScanBatchBuilder scanJobBuilder, BlackDuckRunData blackDuckRunData) {
+        if (signatureScannerOptions.getCsvArchive()) {
+            if (blackDuckRunData != null 
+                    && blackDuckRunData.getBlackDuckServerVersion().isPresent()
+                    && !blackDuckRunData.getBlackDuckServerVersion().get().isAtLeast(MIN_CSV_ARCHIVE_VERSION)) {
+                logger.error("CSV archive was requested but associated Black Duck server is not compatible.");
```

Comment:
Suggestion for clarity around the Black Duck server version being the problem.
"The associated Black Duck server version is not compatible with the CSV archive feature."